packages/desktop-gui/src/settings/experiments.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { observer } from 'mobx-react'
+import ipc from '../lib/ipc'
+import { get } from 'lodash'
+import { parseExperiments } from '@packages/runner/src/lib/experiments'
+
+const openHelp = (e) => {
+  e.preventDefault()
+  ipc.externalOpen('https://on.cypress.io/experiments')
+}
+
+const openIssue = (number) => (e) => {
+  e.preventDefault()
+  ipc.externalOpen(`https://github.com/cypress-io/cypress/issues/${number}`)
+}
+
+const Experiments = observer(({ project }) => {
+  const resolvedEnv = get(project, 'resolvedConfig.env.EXPERIMENTS.value', '')
+  const experiments = parseExperiments(resolvedEnv)
+
+  return (
+    <div>
+      <a href='#' className='learn-more' onClick={openHelp}>
+        <i className='fas fa-info-circle'></i> Learn more
+      </a>
+
+      <div>
+        <p>
+        You can enable some beta features still under development by setting a special environment variable while running
+        Cypress. For example to enable experiments "featureA" and "featureB", open or run Cypress with:
+        </p>
+
+        <pre className='line-nums'>
+          <span>CYPRESS_EXPERIMENTS=featureA,featureB cypress open</span>
+          <span># or</span>
+          <span>CYPRESS_EXPERIMENTS=featureA,featureB cypress run</span>
+        </pre>
+      </div>
+
+      <div>
+        <h3><input type="checkbox" readOnly defaultChecked={experiments.componentTesting} /> component testing</h3>
```

Comment:
i don't know how crazy I am about using a disabled checkbox to distinguish between enabled and disabled - would prefer "Status: Enabled" or "Status: Disabled" instead, for example

Reply:
wait, something is weird, this is missing some of my commits!

Reply:
now it is good!